crates/ruff_linter/resources/test/fixtures/flake8_bandit/S104.py

```diff
@@ -18,3 +18,7 @@ def func(address):
 def my_func():
     x = "0.0.0.0"
     print(x)
+
+
+# Implicit string concatenation
+"0.0.0.0" f"0.0.0.0{expr}0.0.0.0"
```

Comment:
This is not exactly correct (https://github.com/astral-sh/ruff/issues/10308) but for now it's fine.